examples/ssr-hydrate-axum-islands/README.md

```diff
@@ -0,0 +1,90 @@
+<picture>
```

Comment:
Would you mind to simplify the README? See the [axum example](https://github.com/mondeja/leptos-fluent/tree/master/examples/ssr-hydrate-actix) as reference.